frontend/src/components/User/UserEntry/UserEntry.js

```diff
@@ -12,17 +12,18 @@ const UserEntry = (props) => (
                   type={props.type}
                   checked={props.isChecked}
                   onChange={props.checkhandler}
+                  label={props.userName}
                 />
             )
             : <div /> }
-        <h5>{props.userName}</h5>
```

Comment:
I'm afraid this line should be reverted, since if 'props.showCheck' is false then nothing will be shown. My intention was to make the checkbox invisible unless the user clicked the kick off button. 'props.userDesc' is optional, but we need 'props.userName' at here.

Reply:
@njsh4261 Got it.